src/meta-srv/src/bootstrap.rs

```diff
@@ -72,7 +99,7 @@ pub fn router(meta_srv: MetaSrv) -> Router {
         .add_service(admin::make_admin_service(meta_srv))
 }
 
-pub async fn make_meta_srv(opts: MetaSrvOptions) -> Result<MetaSrv> {
+pub async fn build_meta_srv(opts: &MetaSrvOptions) -> Result<MetaSrv> {
```

Comment:
it seems `opts` can still be moved here?

Reply:
I see `opts` is used in many places below, thus we need not (cannot?) remove it I think?